requirements/doc.txt

```diff
@@ -79,7 +79,7 @@ djangorestframework==3.13.1
     # via
     #   -r requirements/base.txt
     #   django-rest-swagger
-doc8==0.11.2
+doc8==1.0.0
```

Comment:
Only major change is drop of support for Python 3.6
https://github.com/PyCQA/doc8/releases/tag/v1.0.0